keras_cv/bounding_box/converters.py

```diff
@@ -170,99 +159,112 @@ def _xyxy_no_op(boxes, images=None, image_shape=None):
 
 
 def _xyxy_to_xywh(boxes, images=None, image_shape=None):
-    left, top, right, bottom, rest = tf.split(
-        boxes, [1, 1, 1, 1, boxes.shape[-1] - 4], axis=-1
-    )
+    left, top, right, bottom = tf.split(boxes, [1, 1, 1, 1], axis=-1)
     return tf.concat(
-        [left, top, right - left, bottom - top, rest],
+        [left, top, right - left, bottom - top],
         axis=-1,
     )
 
 
 def _xyxy_to_rel_xywh(boxes, images=None, image_shape=None):
     image_height, image_width = _image_shape(images, image_shape, boxes)
-    left, top, right, bottom, rest = tf.split(
-        boxes, [1, 1, 1, 1, boxes.shape[-1] - 4], axis=-1
-    )
+    left, top, right, bottom = tf.split(boxes, [1, 1, 1, 1], axis=-1)
     left, right = (
         left / image_width,
         right / image_width,
     )
     top, bottom = top / image_height, bottom / image_height
     return tf.concat(
-        [left, top, right - left, bottom - top, rest],
+        [left, top, right - left, bottom - top],
         axis=-1,
     )
 
 
 def _xyxy_to_center_xywh(boxes, images=None, image_shape=None):
-    left, top, right, bottom, rest = tf.split(
-        boxes, [1, 1, 1, 1, boxes.shape[-1] - 4], axis=-1
-    )
+    left, top, right, bottom = tf.split(boxes, [1, 1, 1, 1], axis=-1)
     return tf.concat(
-        [(left + right) / 2.0, (top + bottom) / 2.0, right - left, bottom - top, rest],
+        [(left + right) / 2.0, (top + bottom) / 2.0, right - left, bottom - top],
         axis=-1,
     )
 
 
 def _rel_xyxy_to_xyxy(boxes, images=None, image_shape=None):
     image_height, image_width = _image_shape(images, image_shape, boxes)
-    left, top, right, bottom, rest = tf.split(
-        boxes, [1, 1, 1, 1, boxes.shape[-1] - 4], axis=-1
+    left, top, right, bottom = tf.split(
```

Comment:
Did the linter expand this to a ton of lines? Seems like it doesn't need to be broken up like this (same for line 213)

Reply:
Same

Reply:
Unfortunately yes

Reply:
If that's the case, can we make [1, 1, 1, 1] as a constant in the header and re-use it here?

Reply:
Done